BFWControls/Modules/Table/Controller/StaticTableViewController.swift

```diff
@@ -78,22 +77,8 @@ class StaticTableViewController: UITableViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        if intrinsicHeightCells {
-            tableView.estimatedRowHeight = 44.0
-        }
```

Comment:
Are you sure you want to remove the `if intrinsicHeightCells` wrapper around this? I expect we want to set `estimatedRowHeight` is `intrinsicHeightCells` is `true`.